pep-0676.rst

```diff
@@ -9,20 +9,22 @@ Content-Type: text/x-rst
 Created: 01-Nov-2021
 Post-History: 23-Sep-2021
 
+
 Abstract
 ========
 
 This PEP addresses the infrastructure around rendering PEP files from
-reStructuredText_ files to HTML webpages. We aim to specify a self-contained
+`reStructuredText`_ files to HTML web pages. We aim to specify a self-contained
 and maintainable solution for PEP readers, authors, and editors.
 
+
 Motivation
 ==========
 
 As of November 2021, Python Enhancement Proposals (PEPs) are rendered in a
-multi-system, multi-stage process. A continuous integration (CI) task runs a
-docutils_ script to render all PEP files individually. The CI task then uploads
-a tar archive to a server, where it is retrieved and rendered into the
+multi-system, multi-stage process. A Continuous Integration (CI) task runs a
```

Comment:
I'd also keep this lowercase, it's a common noun not a proper noun.

```suggestion
multi-system, multi-stage process. A continuous integration (CI) task runs a
```

Reply:
Yeah; the reason I titlecased it in this instance was due to the initialism definition, but I don't see other sources doing that so yeah, you're right.